README.md

```diff
@@ -1,4 +1,31 @@
 # PlaceRecognitionDB
 PlaceRecognitionDB is a tool for creating optimally sized databases (containing the minimum number of frames covering the scene) for place recognition task from RGBD and LiDAR data.
 
-The goal of the project is to create a tool with an optimal compression strategy for any RGBD and LiDAR data.
+The tool supports several basic methods for creating databases, as well as the DominatingSet method for creating optimal databases.
+
+## Datasets format
+To use the tool, your data must be in a specific format.
+* Color images in any format.
+* Depth images corresponding to color images in 16-bit grayscale format or point clouds in `.pcd` format.
+* The trajectory containing one pose in each line in `timestamp (optional) tx ty tz qx qy qz qw` format.
```

Comment:
If the timestamp is optional (by the way, for me it would seems cleaner if the optionality of the timestamp was specified outside of the format string), than how does its presence influence the database creation process? I would assume, that if it is present, than the correspondence between the pose and its color and depth images is determined based on the timestamps, and based on the order of lines and alphabetical order of files otherwise, but I would really prefer it to be specified explicitly (I mean, how is the correspondence between everything determined and what role do the timestamps play)..